File: java/eclipseWS/hello/src/hello/HelloWorld.java
package hello;

public class HelloWorld {
    public static void main(String[] args){
        System.out.print("hello wrold\n");

        System.out.print("args are:\n");
        for(int i=0;i<args.length;i++){
            System.out.println(args[i]);
        }
    }

}

File: gnu/eclipseWS/helloWorld/src/helloWorld.cpp
//============================================================================
// Name        : helloWorld.cpp
// Author      : pangwz
// Version     :
// Copyright   : Your copyright notice
// Description : Hello World in C++, Ansi-style
//============================================================================

#include <iostream>
#include "msg.h"

using namespace std;

int main() {
	cout << "!!!Hello World  asdf!!!" << endl; // prints !!!Hello World!!!

	msg::getInstance()->printMsg();

//	msg* myMsg = new msg(); 			//error! for private construct

	msg::delInstance();
	return 0;
}
